packages/vars/src/typography.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --font-family-system: Objektiv, sans-serif, 'Helvetica Neue', Helvetica, Arial;
+  --font-family-styrene: Objektiv, sans-serif, 'Helvetica Neue', Helvetica,
```

Comment:
what does `styrene` mean?

Reply:
Thanks for noticing that. But actually, the css-variables package has not been finished yet. I thought to use it as an intermediate variable for the font, but I've checked, and it's not needed now.
I'll replace it.